styles.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+#back-images-intro {
+  width: 100%;
+  height: 100%;
+  padding-top: 24px;
+  background-image: url('./images/header-shapes\ mobile.svg');
+}
+
+.connectlink {
+  padding: 12px;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  color: #7f8cff;
+}
+
+h1 {
+  padding-left: 12px;
+  width: 327px;
+  height: 150px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding-top: 206px;
+  color: #172b4d;
+}
+
+header {
```

Comment:
- [ ] [Optional] For best practices it would be better if you select the element using classes instead of tag names. This is optional but highly recommended